### DISCUSSION TITLE
Is announce.dismiss compatible with Instant loading?

### DISCUSSION THREAD

**Discussion by Stephen-Gates** [squidfunk/mkdocs-material#3300] [Q&A] [answered]
When I use announce.dismiss with Instant loading enabled, the announcement is only temporarily dismissed on each page. 

If I disable the Instant loading feature, announce.dismiss works ok but I sometimes have to dismiss the announcement across pages. Browser setting issue?  

Is announce.dismiss compatible with Instant loading, or are they mutually exclusive?

**Comment by squidfunk** [comment, marked as answer]
Clearly a bug, please file an issue! 😅

> If I disable the Instant loading feature, announce.dismiss works ok but I sometimes have to dismiss the announcement across pages. Browser setting issue?

I need to know exactly what you're doing. Note that the announcement is dismissed as long as the content is the same, so if you're using different templates with different announcements, that won't work. Also, the URL has to be the same, as local storage is used.

**Reply by squidfunk** [reply]
Turns out there were some other issues with instant loading and the storage helpers which I just fixed in 632eff2e1 . It's in the category of "how did that ever work", but we didn't hit these behaviors until now, as all functions/components using it were independent of instant loading. The announcement bar however is. I'm pushing these changes over to Insiders in a moment.

**Reply by squidfunk** [reply]
... oh and I could reduce the size of the helper script from 350 to 270 characters, so definitely worth the small refactor 😊

**Reply by squidfunk** [reply]
It's fixed on `master`. I'll issue a new release today.

**Reply by squidfunk** [reply]
Released as part of **8.0.4+insiders-4.3.0** 😊

**Comment by Stephen-Gates** [comment]
I'm not sure that solved the problem. Using:

```
  custom_dir: overrides
  features:
  - announce.dismiss
  - navigation.instant
  ```

And changing the `{% block announce %}` entry, when I publish the announcement quickly flashes in the display. Then when I go to other pages the announcement doesn't show.

**Reply by squidfunk** [reply]
I'm not sure I understand?
